### ISSUE TITLE
Initializing a Bazel project

### ISSUE THREAD

**Issue by nc0fr** [nc0fr/NC0#7]
[Bazel](https://bazel.build) is an amazing polyglot and powerful build system with remote execution properties to make building projects quicker (such as Rust projects!!!!).

This ticket exists to debate about whether or not to use the new MODULE system versus the traditional WORKSPACE setup (noting that the Bazel team is pushing the MODULE system over the long run).

**Comment by RealHinome** [comment]
For long time support, you should use MODULE. Since v7, Bazel is creating MODULE file when trying to compile and, as you said, they are forcing the usage of MODULE instead of WORKSPACE. In the v9, it'll be totally deleted; no more support.
This is my personal opinion.

**Comment by nc0fr** [comment]
You're right, my only issue is with the poor support of Bzlmod from many public rules yet.
But either way, since dependencies will most likely be vendored using Copybara, using the WORKSPACe system or the MODULE system would simply be a matter of writing `local_repository` or `local_module` 🤷.
But for sure, the MODULE system seems to provide a way more naive and simple flow.
